sdks/java/container/boot.go

```diff
@@ -115,6 +115,7 @@ func main() {
 	os.Setenv("PIPELINE_OPTIONS", options)
 	os.Setenv("LOGGING_API_SERVICE_DESCRIPTOR", proto.MarshalTextString(&pipepb.ApiServiceDescriptor{Url: *loggingEndpoint}))
 	os.Setenv("CONTROL_API_SERVICE_DESCRIPTOR", proto.MarshalTextString(&pipepb.ApiServiceDescriptor{Url: *controlEndpoint}))
+	os.Setenv("RUNNER_CAPABILITIES", strings.Join(" ", info.GetRunnerCapabilities()))
```

Comment:
https://golang.org/pkg/strings/#Join
You've swapped the parameter order to strings.Join, the join separator is 2nd.
```suggestion
	os.Setenv("RUNNER_CAPABILITIES", strings.Join(info.GetRunnerCapabilities(), " "))
```